src/lib.rs

```diff
@@ -49,7 +47,6 @@ static ESPLORA_TIMEOUT_SECS: u64 = 30;
 #[allow(dead_code)]
 pub struct LightningNode {
     #[allow(dead_code)]
-    rt: AsyncRuntime,
```

Comment:
**_Hold on, not so reckless!_** We will need the async runtime for the background processor.

Reply:
Oh. I force-pushed a new version, which purges a lot less aggressively.